libp2p/protocols/pubsub/gossipsub/scoring.nim

```diff
@@ -250,43 +250,43 @@ proc punishInvalidMessage*(g: GossipSub, peer: PubSubPeer, msg: Message) {.async
         await g.disconnectPeer(peer)
         raise newException(PeerRateLimitError, "Peer disconnected because it's above rate limit.")
 
+  let topic = msg.topic
+  # TODO: REVIEW
+  if topic notin g.topics:
```

Comment:
even if the topic is not in topics, `invalidMessageDeliveries` needs to be incremented

Reply:
But that was not what was happening before:
```nim
for tt in msg.topicIds:
    let t = tt
    if t notin g.topics:
      continue

    let tt = t
    # update stats
    g.withPeerStats(peer.peerId) do (stats: var PeerStats):
      stats.topicInfos.mgetOrPut(tt, TopicInfo()).invalidMessageDeliveries += 1
```
Only topics in `g.topics` are counted in `invalidMessageDeliveries`. 
Was the previous code wrong?

Reply:
you're right - I misread the indent level

Reply:
let's get rid of this TODO then

Reply:
Done.